keras_contrib/callbacks/dead_relu_detector.py

```diff
@@ -16,16 +15,22 @@ class DeadReluDetector(Callback):
             True means that even a single dead neuron triggers warning
             False means that only significant number of dead neurons (10% or more)
             triggers warning
+        bool_warning: output mode
+            True means a warning is raised
+            False means the warning message is printed.
     """
-    def __init__(self, x_train, verbose=False):
+
+    def __init__(self, x_train, verbose=False, bool_warning=False):
```

Comment:
can this become a string option with the variable name `on_detection` where accepted parameters are `'raise'` and `'print'`?

Reply:
Do you think it is a good idea that the training should fail when a neuron is dead? If you wan't to prevent the neurons from dying I get that. But than we also have to be sure that this is checked before the network is saved.
At this point the printing works (with testing), I might update in a few minutes.

Reply:
Yes the other option is make this printing only, that's ok with me I'd be happy to merge if the param was gone entirely too.

Reply:
I believe it might be a good idea to do something if a condition is met (e.g. a dead neuron). However this is more general. Do we agree to keep it to a print only for now?

Reply:
yep print is good. Thinking this through some more... Is there already a return value or some callback that lets the user detect the problem and act on it in their code? 

I don't think a print alone would let people detect a failure and, for instance, restart training in an automated fashion.